ops/services/alerts/app_service_metrics/uptime.tf

```diff
@@ -0,0 +1,74 @@
+locals {
+  geo_locations = [
+    "us-va-ash-azr",   // East US
+    "us-ca-sjc-azr",   // West US
+    "us-il-ch1-azr",   // North Central US
+    "us-tx-sn1-azr",   // South Central US
+    "apac-jp-kaw-edge" // Japan (Guam)
+  ]
+
+  url_prefix = var.env == "prod" ? "www" : var.env
+
+  url_map = {
+    "${var.env}-simplereport-gov"     = "https://${local.url_prefix}.simplereport.gov/",
+    "${var.env}-simplereport-gov-api" = "https://${local.url_prefix}.simplereport.gov/api/actuator/health",
+    "${var.env}-simplereport-gov-app" = "https://${local.url_prefix}.simplereport.gov/app/health/ping"
+  }
+}
+
+resource "azurerm_application_insights_web_test" "uptime" {
+  for_each = local.url_map
+
+  name                    = each.key
+  description             = "Verify the URL is publicly available"
+  location                = var.rg_location
+  resource_group_name     = var.rg_name
+  application_insights_id = var.app_insights_id
+  retry_enabled           = true
+  frequency               = 300
+  timeout                 = 60
+  geo_locations           = local.geo_locations
+  kind                    = "ping"
+  enabled                 = contains(var.disabled_alerts, "uptime") ? false : true
+
+  // Borrowed from https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_insights_web_test#example-usage
+  configuration = <<XML
+<WebTest Name="WebTest1" Id="" Enabled="True" CssProjectStructure="" CssIteration="" Timeout="120" WorkItemIds="" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010" Description="" CredentialUserName="" CredentialPassword="" PreAuthenticate="True" Proxy="default" StopOnError="False" RecordedResultFile="" ResultsLocale="">
+  <Items>
+    <Request Method="GET" Guid="" Version="1.1" Url="${each.value}" ThinkTime="0" Timeout="300" FollowRedirects="True" RecordResult="True" Cache="False" ResponseTimeGoal="0" Encoding="utf-8" ExpectedHttpStatusCode="200" ExpectedResponseUrl="" ReportingName="" IgnoreHttpStatusCode="False" />
```

Comment:
Not sure what the timeouts mean, but they seem awfully generous.

Reply:
This one's weird because you're forced to use an XML config block even though you're also setting some of these values in the TF resource, so it's unclear which `timeout` value applies. Agreed that `300` is way too high - I'm going to set them all to `30`, which is the default value for the TF resource.